.github/workflows/conda-package.yml

```
@@ -209,12 +210,19 @@ jobs:
           python -c "import dpnp; print(dpnp.__version__)"
 
       - name: Run tests
-        if: env.RERUN_TESTS_ON_FAILURE != 'true'
+        if: env.RERUN_TESTS_ON_FAILURE != 'true' && matrix.python != env.LATEST_PYTHON
         run: |
           python -m pytest -n auto -ra --pyargs ${{ env.PACKAGE_NAME }}.tests
 
+      - name: Run tests for all dtypes
+        if: env.RERUN_TESTS_ON_FAILURE != 'true' && matrix.python == env.LATEST_PYTHON
+        env:
+          DPNP_TEST_ALL_INT_TYPES: 1
+        run: |
+          python -m pytest -ra --pyargs ${{ env.PACKAGE_NAME }}.tests
```

Would it be better to export `DPNP_TEST_ALL_INT_TYPES` based on python used rather then duplicate the steps, something like:
```suggestion
          if [[ ${{ matrix.python }} -eq ${{ env.LATEST_PYTHON }} ]]; then
              export DPNP_TEST_ALL_INT_TYPES=1
          fi

          python -m pytest -ra --pyargs ${{ env.PACKAGE_NAME }}.tests
```